source/extensions/omni.isaac.lab/omni/isaac/lab/assets/articulation/articulation.py

```diff
@@ -24,14 +24,14 @@
 import omni.isaac.lab.utils.string as string_utils
 from omni.isaac.lab.actuators import ActuatorBase, ActuatorBaseCfg, ImplicitActuator
 
-from ..rigid_object import RigidObject
+from ..asset_base import AssetBase
 from .articulation_data import ArticulationData
 
 if TYPE_CHECKING:
     from .articulation_cfg import ArticulationCfg
 
 
-class Articulation(RigidObject):
+class Articulation(AssetBase):
     """An articulation asset class.
```

Comment:
Can you do a Ctrl-F for any instances of rigid + object and remove / update them? The docstring has some references to it being a subclass

Reply:
Done.